src/app.js

```diff
@@ -96,6 +97,8 @@ client.on("message", (msg) => {
     foaas(msg);
   } else if (msg.content.toLowerCase().match(/^!dailyarticles/g)) {
     devToArticles(newsChannel);
+  } else if (msg.content.toLowerCase().match(/hail hydra/g)) {
+    hailHydra(msg);
```

Comment:
Getting to be switch territory. Good for now. Might be a good readability idea to 

```js
const message = msg.content.toLowerCase();
const FU_JSON_REGEX = /^!fujson/g;
...

if(message.match(FU_JSON_REGEX) {
etc etc
```